Makefile

```diff
@@ -350,8 +350,7 @@ lint-prettier: build-formatter ## Run ts/js/yaml lint checks
 
 .PHONY: lint-hcl
 lint-hcl: ## Check to see if Packer templates are formatted properly
-	.buildkite/scripts/lint_packer.sh
-	.buildkite/scripts/lint_nomad.sh
+	docker-compose --file=docker-compose.check.yml run --rm hcl-lint
```

Comment:
My preference is that we always run with `--user` and never run containers as root, regardless of whether we're writing files back to the host or not. But I'm fine with it for now.

Reply:
I'd previously tried enforcing this by putting `user: ${UID}:${GID}` in compose files, both of which are set in the Makefile.

Reply:
BTW, is this PR missing a def for `COMPOSE_USER` in the Makefile?